### ISSUE TITLE
pair sims are showing bias

### ISSUE THREAD

**Issue by esheldon** [esheldon/metadetect#130]
For separation 1.35 arcsec I see bias with and without deblending

no deblending
```
0.00475472 < m1 < 0.0135739
```
with deblending
```
0.0635627 < m1 < 0.0767297
```

**Comment by esheldon** [comment]
At 4.0 arcsec things look ok
```
-5.47526e-05 < m1 < 0.0011639 (99.7%)
```
all results are for wmom

**Comment by beckermr** [comment]
I assume this is for lsst?

**Comment by esheldon** [comment]
Yeah, I've tested this with regular metadetect that uses sep and it has much better accuracy.

**Comment by esheldon** [comment]
I'm seeing a bias running sx on these sims as well.  So it has something to do with the simulation data produced

**Comment by beckermr** [comment]
> forgot to mention that, with the deblending, the R value is actually negative

Do you see this with sx?

**Comment by esheldon** [comment]
No, the R value looks normal for sx and for lsst without deblending

**Comment by beckermr** [comment]
What bias do you see with SX? I don't think we expect mdet to be perfect for pairs near where the detection rate for two objects is 50%. In a real universe we get to average over many separations.

**Comment by esheldon** [comment]
0.0024349 < m1 < 0.00695481 (99.7%)

with my other sims I saw a maximum m of about 0.0005

**Comment by beckermr** [comment]
> with my other sims I saw a maximum m of about 0.0005

Which other sims?

**Comment by esheldon** [comment]
There was the code from the metadetect paper, you know those sims.

I also reimplemented that recently for the pair tests I showed you in slack.  But these sims are so simple I doubt there was any substantial differences between those two implementations.

substantial differences in the sims

1. pixel scale change from 0.263 to 0.2
2. I used moffat psf for the other sims, gaussian for these
3. the other sims didn't try to use postage stamps, rather the entire scene was drawn at once

**Comment by esheldon** [comment]
Also these galaxies are smaller relative to the psf `hlr/psf_fwhm` of 0.6 rather than 1.1 for the other sims

**Comment by esheldon** [comment]
I think the bias difference is mainly due to the smaller size of the objects used in the newer sims, hlr=0.5 vs hlr=1.0

I ran the old sim code with small galaxies and I also see a bias.

Here is sx, hlr=0.5, fixed separation 1.35 arcsec but as a function of the input shear (note the sims that started this thread were at 0.02)

![m-vs-shear](https://user-images.githubusercontent.com/621661/134563940-249babc4-d2ae-4d64-97f0-6123fca38e25.png)
